ArduPlane/quadplane.cpp

```diff
@@ -701,45 +698,44 @@ bool QuadPlane::setup(void)
 
     const static char *strUnableToAllocate = "Unable to allocate";
     if (!motors) {
-        hal.console->printf("%s motors\n", strUnableToAllocate);
-        goto failed;
+        AP_BoardConfig::config_error("%s motors", strUnableToAllocate);
```

Comment:
Could we flip this around to be:
```suggestion
        AP_BoardConfig::config_error("Unable to allocate %s", "motors");
```

The compiler will coalesce the string constants.